src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -706,8 +825,11 @@ internal override void OnKeyDown(SDL.SDL_KeyboardEvent e)
             if (TargetManager.IsTargeting && e.keysym.sym == SDL.SDL_Keycode.SDLK_ESCAPE && Keyboard.IsModPressed(e.keysym.mod, SDL.SDL_Keymod.KMOD_NONE))
                 TargetManager.CancelTarget();
 
-            if (!Engine.UI.IsKeyboardFocusAllowHotkeys)
-                return;
+            if (Engine.Profile.Current.ActivateChatAfterEnter)
```

Comment:
This is a revert of Mel's quite amazing keyboard focus improvements?

Reply:
Might be, I can revise this.